.storybook/helpers/TokenTable.jsx

```diff
@@ -46,16 +46,15 @@ const previewRenderers = {
  *
```

Comment:
This helper and all of its consumers (design token stories) are a bit of an odd setup. 

My goal here is to preserve the functionality until we use a dedicated plugin or tool to render design tokens that come from the UI refresh